src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: 'Under construccion',
```

Comment:
- [ ] Here we do not need to `status` in the initialState, Kindly remove it.

Reply:
Hi @Deepakdanger, thanks for your feedback.